avm/res/elastic-san/elastic-san/tests/common.tests.ps1

```diff
@@ -179,7 +179,6 @@ function Test-VerifyElasticSANPrivateEndpoints($GroupIds, $PrivateEndpointConnec
 
             if ($PrivateEndpointCounts -ne 0) {
                 $PrivateEndpoints[$i].name | Should -Not -BeNullOrEmpty
-                $PrivateEndpoints[$i].location | Should -Not -BeNullOrEmpty
```

Comment:
Should we have tests for common types at module level? Wondering as I fear it may become difficult to maintain them. Not blocking for this PR, just food for thoughts

Reply:
As said, just food for thoughts. Resolving